R/download.R

```diff
@@ -1,46 +1,3 @@
-#' @title Download ACS
-#' @description Passes in a list of parameters to download the ACS census
-#' data using functions from the tidycensus package
-#' @param tbls list. List of census tables with attributes to download
-#' @param raw_pth. Path to save the raw data.
-#' @details To find a list of parameters to pass see documentation for 
-#' tidycensus::get_acs
-#' @examples 
-#' \dontrun{
-#' tbls <- list(B23008 = list(year = 2019, state = 48, 
-#'                                geography = "tract", county = 439))
-#' raw_pth <- "C:/"
-#' dwnld.acs(tbls = tbls, raw_pth = raw_pth)
-#' }
-dwnld.acs <- function(tbls, raw_pth, ...) {
```

Comment:
I moved this function to acs_dm because it seems to make more sense to have it with the rest of the data management steps, rather than in it's own file.